bundles/org.openhab.binding.insteon/src/main/resources/OH-INF/thing/channels.xml

```diff
@@ -552,8 +552,14 @@
 
 	<channel-type id="relay-sensor-follow" advanced="true">
 		<item-type>Switch</item-type>
-		<label>Output Relay Sensor Follow</label>
-		<description>Set the output relay to trigger when the sensor input changes state.</description>
+		<label>Relay Sensor Follow</label>
```

Comment:
Maybe very minor, but if already creating upgrade instructions, this channel should also be listed.